app/models/miq_provision_request_template.rb

```diff
@@ -1,8 +1,10 @@
 class MiqProvisionRequestTemplate < MiqProvisionRequest
   def create_tasks_for_service(service_task, parent_svc)
     template_service_resource = ServiceResource.find_by_id(service_task.options[:service_resource_id])
-    scaling_min = template_service_resource.try(:scaling_min) || 1
+    scaling_min = number_of_vms(service_task, parent_svc, template_service_resource)
+    scaling_min ||= template_service_resource.try(:scaling_min) || 1
 
+    _log.info("create_tasks_for_service ID #{service_task.id} SCALING : #{scaling_min}")
```

Comment:
@mkanoor 
Since scaling isn't currently used with services, should we change the log message from SCALING to Number Of VMs?